### PR TITLE
Please Review: US States

### PR DESCRIPTION
**Pull request by spolu**

### CODE REVIEW

File: spec/country_reverse_geocoding.spec.js [hunk 1]
```diff
@@ -22,7 +22,39 @@ describe("Country reverse geocoding", function() {
 
   it("should return null if no country was matched", function() {
     var results = crg.get_country(55.5, -30.9);
+    expect(results).toBe(null);
+  });
+
+  it("should return information about a us state", function() {
+    var results = crg.get_us_state(32.377447, -86.300942);
+    console.log(results);
```
Comment (gflandre) [inline spec/country_reverse_geocoding.spec.js]: :boom:

File: spec/country_reverse_geocoding.spec.js [hunk 2]
```diff
@@ -22,7 +22,39 @@ describe("Country reverse geocoding", function() {
...
+
+    expect(typeof results).toBe('object');
+    expect(typeof results.code).not.toBe('undefined');
+    expect(typeof results.name).not.toBe('undefined');
+    // This particular point should be in France
```
Comment (gflandre) [inline spec/country_reverse_geocoding.spec.js]: in Alabama

**Comment by spolu** [comment]
fixed

**Comment by gflandre** [comment]
OK

**Comment by n1t0** [comment]
OK

**Pull request merged**